packages/core/data-transfer/src/strapi/providers/local-destination/index.ts

```diff
@@ -78,6 +82,31 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
     return restore.deleteRecords(this.strapi, this.options.restore);
   }
 
+  async #deleteAllAssets() {
+    assertValidStrapi(this.strapi);
+
+    if (strapi.config.get('plugin.upload').provider === 'local') {
+      const uploadsDirectory = path.join(
+        this.strapi.dirs.static.public,
+        this.uploadsBackupDirectoryName
```

Comment:
What if the backup directory name is ''? Will it try to delete the upload folder itself?